osu.Game/Screens/Play/HUD/LegacyComboSplash.cs

```diff
@@ -0,0 +1,206 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Collections.Generic;
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Graphics.Textures;
+using osu.Game.Configuration;
+using osu.Game.Rulesets.Scoring;
+using osu.Game.Skinning;
+using osuTK;
+
+namespace osu.Game.Screens.Play.HUD
+{
+    public class LegacyComboSplash : Container<Container<Sprite>>, ISkinnableDrawable
```

Comment:
This generic type (`Container<Container<Sprite>>`) is a bit much. Is this just for the test? If so, I'd recommend switching over to `.ChildrenOfType<>()` there.